packages/dd-trace/src/llmobs/plugins/base.js

```diff
@@ -43,7 +43,7 @@ class LLMObsPlugin extends TracingPlugin {
       llmobsStorage.enterWith({ span })
       ctx.llmobs.parent = parent
 
-      this._tagger.registerLLMObsSpan(span, { parent, ...registerOptions })
+      this._tagger.registerLLMObsSpan(span, { parent, integration: this.constructor.id, ...registerOptions })
```

Comment:
we'll also want to add this [here](https://github.com/DataDog/dd-trace-js/blob/2c5e17c2808bac1ec89612c4fd11893d21fb9ce1/packages/dd-trace/src/llmobs/plugins/bedrockruntime.js#L53-L59) - we had to special case bedrock bc the way its instrumented doesn't use the nice tracing channel events (start, end, asyncEnd, etc.) but instead custom event names. should just be able to hardcode `bedrock` there (or feel free to set it up as a constant!)